src/styles/list.scss

```diff
@@ -71,8 +71,13 @@
 .ob-list__text-tertiary,
 .ob-list__text-secondary {
   font-size: 0.8rem;
+}
+.ob-list__text-tertiary {
   color: #757575;
```

Comment:
$grey is #757575 sorry

Reply:
Essentially the task is to put it back how it was

Reply:
Got it